server/rest/src/main/java/org/infinispan/rest/RestServer.java

```diff
@@ -111,7 +111,6 @@ protected void startInternal(RestServerConfiguration configuration, EmbeddedCach
 
    @Override
    public int getWorkerThreads() {
-      // Unused for now, so just return the smallest possible valid value
-      return 1;
+      return Integer.getInteger("infinispan.server.rest.workerThreads", configuration.workerThreads());
```

Comment:
I created https://issues.jboss.org/browse/ISPN-10371 to redo this and all the server protocol thread pools as they are extraneous for what we want in 10.0.

Reply:
Yes, thanks, I was about to open a JIRA, since it was out of the scope of this PR